test/AssetPool.test.js

```diff
@@ -194,6 +194,8 @@ describe("AssetPool", () => {
       const allocatedReward = to1e18(70)
 
       beforeEach(async () => {
+        await increaseTime(86400) // 1 day
```

Comment:
This is not needed here, or is it?

Reply:
eh, not sure how it got there... 83048dd